### DISCUSSION TITLE
Background color for selection

### DISCUSSION THREAD

**Discussion by tupo2** [geany/geany#3836] [Q&A] [answered]
Hello,

When a text is selected and the editor is unfocused the background color becomes #808080.
Can it be altered?

**Comment by elextr** [comment, marked as answer]
The Scintilla editing widget Geany uses only recently added the capability with the new [element API](https://www.scintilla.org/ScintillaDoc.html#ElementColours), but Geany does not expose it, well written PRs that integrate it into colour schemes are welcome.
